src/composables/QuasarFieldRule.js

```diff
@@ -58,3 +58,77 @@ export function isUniqueFileLabel(t, tree, value) {
   return tree.every(({ label }) => label !== value)
     || t('errors.fileExplorer.label.duplicate');
 }
+
+/**
+ * Check if the value is required.
+ * @param {Function} t - I18n translate function.
+ * @param {String|Number|Boolean} value - Value to check.
+ * @param {Boolean} required - Rules value.
+ * @return {Boolean|String} Return true if value is required and exist,
+ * otherwise the translated error message.
+ */
+export function isRequired(t, value, required) {
+  return !required
+    || !(value === null || value === undefined || value === '')
+    || t('errors.rules.required');
+}
+
+/**
+ * Check if the value length is superior or equal to min.
```

Comment:
`superior` -> `greater than` for consistency with translation

Reply:
done